index.html

```diff
@@ -16,6 +16,18 @@
     <title>Portfolio</title>
   </head>
   <body>
+  <!--Mobile menu-->
+
+  <div id="menuId" class="myMenu">
+    <ul >
+      <li class="claList"><a href="#portfolio">Portfolio</a></li>
+      <li class="claList"><a href="#about">About</a></li>
+      <li class="claList"><a href="#contact">Contact</a></li>
+    </ul>
+    <img src="./image/IconX.svg" alt="Close Menu" class="close" id="closeMenu">
+  </div>
```

Comment:
- [ ] When I click the hamburger nothing happens. It should open the menu section and after opening, if I click on any item/link from hamburger it should close it. Please work on the feature.